<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dictionaryGUI</class>
 <widget class="QMainWindow" name="dictionaryGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dictionaryGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Select your dictionary and text file:</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QTextEdit" name="misspelledWordOccurances">
      <property name="autoFormatting">
       <set>QTextEdit::AutoNone</set>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QTextEdit" name="fileDisplayed"/>
    </item>
    <item row="11" column="1">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Occurances : Misspelled Word</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="NumberofPopularWords">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Number of Popular Words:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="CorrectWordsIndex">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Occurances : Correct Word</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="saveAsButton">
      <property name="text">
       <string>Save As</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="exitPushButton">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="dictionarySelection">
      <item>
       <property name="text">
        <string>Choose Dictionary</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>British</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>American</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QSpinBox" name="numberOfPopularSpinBox">
      <property name="maximum">
       <number>999999</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="openButton">
      <property name="text">
       <string>Open File</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QTextEdit" name="foundWordOccurances">
      <property name="font">
       <font>
        <family>Ubuntu</family>
       </font>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>label</zorder>
   <zorder>foundWordOccurances</zorder>
   <zorder>fileDisplayed</zorder>
   <zorder>dictionarySelection</zorder>
   <zorder>exitPushButton</zorder>
   <zorder>openButton</zorder>
   <zorder>NumberofPopularWords</zorder>
   <zorder>numberOfPopularSpinBox</zorder>
   <zorder>misspelledWordOccurances</zorder>
   <zorder>CorrectWordsIndex</zorder>
   <zorder>label_5</zorder>
   <zorder>saveAsButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew_Window">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="fileopen"/>
    <addaction name="actionSave_As"/>
   </widget>
   <addaction name="menuNew_Window"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <kerning>false</kerning>
    </font>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionNew_Window"/>
  </widget>
  <action name="actionNew_Window">
   <property name="text">
    <string>Selection Window</string>
   </property>
  </action>
  <action name="fileopen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>exitPushButton</sender>
   <signal>clicked()</signal>
   <receiver>dictionaryGUI</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
